### DISCUSSION TITLE
Waiting call

### DISCUSSION THREAD

**Discussion by PedroMPagani** [xetorthio/jedis#3216] [Q&A]
When does a Jedis connection becomes broken?

Currently I am having a specific own jedis pool for thousands of publish, but I don't wanna keep the wait from the <Long> response, just kind of ignoring and just calling the .sendMessage, but to be most certain that these messages will be delivered, when does it becomes broken? mainly what cases, if not pinging the redis server enough or what

**Comment by sazzad16** [comment]
1. Network issue.
2. Improper usage in multi-thread.
